src/Program.cs

```diff
@@ -81,13 +82,14 @@ public static async Task<int> Run(string workspace, string verbosity, bool dryRu
 
                 Build.Locator.MSBuildLocator.RegisterInstance(msBuildInstance);
 
-                return await CodeFormatter.FormatWorkspaceAsync(
+                var formatResult = await CodeFormatter.FormatWorkspaceAsync(
                     logger,
                     workspacePath,
                     isSolution,
                     logAllWorkspaceWarnings: logLevel == LogLevel.Trace,
                     saveFormattedFiles: !dryRun,

```

Comment:
Does running a `--check` imply that `dryRun` should also be true or would you expect `--check` to update files on disk?

Reply:
The `--check` option will affect only program **exit code**. All other options, even if combined with `--check` options, will continue to work in the same manner as worked without my PR. I just tried to make minimal working implementation in my PR which will solve continuous integration scenarios.